File: CinemaProject.WebAPI/CinemaProject.BLL/Models/Hall.cs
using System;

namespace CinemaProject.BLL.Models
{
    public class Hall
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Guid CinemaId { get; set; }

        public Seat[] Seats { get; set; }
    }
}

File: CinemaProject.Angular/src/app/AdminPages/Models/hall.ts
class Hall {
  constructor(
    public id: string,
    public name: string,
    public cinemaId: string
  ) {}
}

class HallView {
  constructor(
    public id: string,
    public name: string,
    public cinemaId: string,
    public cinemaName: string
  ) {}
}

export { Hall, HallView };
